test/mithril.prop.js

```diff
@@ -42,6 +42,22 @@ describe("m.prop()", function () {
 		expect(JSON.stringify(obj)).to.equal('{"prop":"test"}')
 	})
 
+	it("correctly stringifies Date", function () {
+		var prop = m.prop(new Date(999))
+		expect(JSON.stringify(prop)).to.equal('"1970-01-01T00:00:00.999Z"')
+	})
+
+	it("correctly stringifies object with toJSON method", function () {
+		function Thing(name) {
+			this.name = name
+		}
+		Thing.prototype.toJSON = function() {
+			return JSON.stringify({kind: 'Thing', name: this.name})
```

Comment:
I think it's easier to read if you just do this
```javascript
return {kind: 'Thing', name: this.name}´
```
than you can revert this quote spamming in the expect.